src/InsertData.py

```diff
@@ -17,18 +17,18 @@
 from pymongo import errors as mongoerrors
 from pathlib import Path
 
-logging.basicConfig(filename = '../log/' + 
-                str(datetime.datetime.now()).replace(' ', '_').replace(':', '')[:17] + '.log', 
-                level=logging.INFO, 
-                format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+logging.basicConfig(filename='../log/' +
```

Comment:
I am thinking about we should have logging module... what do you think?

Reply:
+1

Reply:
yes, that would be cool. maybe we can do it in the next PR?